### PR TITLE
Improve performance

### PR DESCRIPTION
**Pull request by yunfeiguo**
- [ ] [RES-XXXX](https://binatechnologies.atlassian.net/browse/RES-XXXX)
- [ ] Blocking PRs: ADD HERE IF ANY
* People
  - [ ] Reviewers (>=1): ADD HERE
  - [ ] Merger (>=1): ADD HERE
  - [ ] FYI: ADD HERE IF ANY
- [ ] Release note (if needed)
- [ ] No code copied from outside Bina 
- [ ] Tests (Unit, Workflow, ITL & Gherkin) are passing
- [ ] PR labels, milestones & assignees

# Summary

* minor performance improvement for parsing VCF: ~30% (assuming reading from gzipped stream costs constant time).
* ~~TODO: variant sampling performance improvement pending~~
* refactored `build.sh` and `quickstart.sh`

### CODE REVIEW

File: src/main/java/com/bina/varsim/types/Sequence.java [hunk 1]
```diff
@@ -32,6 +32,29 @@ public Sequence(String header, byte[] seq, int len) {
...
     }
 
+    /**
+     * check if the sequence is normal (consisting of normal characters)
+     * empty sequence is considered false
+     * @param s
+     * @return
+     */
+    public static boolean isNormalSequence(final String s) {
+        if (s.length() == 0) {
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/types/Sequence.java]: `s == null`?

File: src/main/java/com/bina/varsim/types/VCFInfo.java [hunk 2]
```diff
@@ -14,21 +16,46 @@
 public class VCFInfo {
     final private Map<String, VCFInfoElement> info2Value;
 
+    /**
+     * compare speed of different string splitting methods
```
Comment (marghoob) [inline src/main/java/com/bina/varsim/types/VCFInfo.java]: Do we know the outcome of this benchmark?

File: src/main/java/com/bina/varsim/types/Sequence.java (hunk 1 above)
Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/types/Sequence.java]: right, thanks!

File: src/main/java/com/bina/varsim/types/VCFInfo.java (hunk 2 above)
Comment (yunfeiguo) [reply src/main/java/com/bina/varsim/types/VCFInfo.java]: just added (for one repeat)

**Comment by marghoob** [comment]
👍

**Pull request merged**